guide/onboarding/automatic-cloud-backup.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: guide
+title: Automatic cloud backup
+description: How to backup a recovery phrase automatically
+grand_parent: Onboarding
+parent: Backing up a recovery phrase
+nav_order: 1
+permalink: /guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup/
+main_classes: -no-top-padding
+---
+
+# Automatic cloud backup
+
+### How it Works
```

Comment:
Switch headers on this page to sentence-case for consistency with the rest of the guide.